workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -364,7 +364,8 @@ private bool TryGetEntityFromRequestId(uint requestId, string responseName, out
             }
             else if (!view.TryGetEntity(entityId, out entity))
             {
-                Debug.LogWarningFormat(TranslationErrors.CannotFindEntityForWorldCommandResponse, entityId, responseName);
+                Debug.LogWarningFormat(TranslationErrors.CannotFindEntityForWorldCommandResponse, entityId,
```

Comment:
If you could rebase onto master and use the new logging stuff, that'd be ideal :)

Reply:
Merged in master.